styles.css

```diff
@@ -493,6 +493,116 @@ header.active .navigations {
   left: 0;
 }
 
+.pop {
+  display: none;
+  position: fixed;
+  margin: 0 auto;
+  top: 0;
+  bottom: 0;
+  max-height: 100vh;
+  width: 100vw;
+  overflow: auto;
+  background: rgb(193 199 208 / 50%);
+  backdrop-filter: blur(3px);
+  padding: 50px 16px;
+}
+
+.modal {
+  max-width: 80%;
+  height: 90%;
+  margin: auto;
+  background-color: white;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  border-radius: 0.7rem;
+}
+
+.overlay-image-container {
+  width: 100%;
+  height: 60%;
+  object-fit: cover;
+  margin-bottom: 2%;
+}
+
+.second-paragraph {
+  display: block;
+}
+
+.cross {
+  border: none;
+  font-size: 2rem;
+}
+
+.liveIcon:hover {
+  color: white;
+}
+
+.pop-content {
+  width: 100%;
+  margin: 0 auto;
+  height: 100vh;
+  background-color: #fff;
+  border: 1px solid #ebecf0;
+  box-sizing: border-box;
+  padding: 20px 16px;
+  border-radius: 15px;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
```

Comment:
- [ ] Kindly consider using other height value or other [height properties](https://css-tricks.com/almanac/properties/h/height/#:~:text=The%20height%20property%20in%20CSS,%3A%20%2D100px%20are%20not%20accepted.) to ensure that in the mobile version of the popup, all the contents of the popup are covered. This will make your popup more professional and user-friendly 👍.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/233770102-bdd17a67-d8f5-48e2-8bdf-67a6ca49eccc.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/233770196-8a4f307b-a0ac-493f-84a8-5bcd181d941c.png)

Design at previous reviewer level 😎 
![image](https://user-images.githubusercontent.com/99394374/233770242-15610054-738f-414b-b775-9d4e43b4c107.png)